oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/authentication/JwtIssuerAuthenticationManagerResolver.java

```diff
@@ -160,8 +199,16 @@ public String convert(@NonNull HttpServletRequest request) {
 
 		private final Predicate<String> trustedIssuer;
 
+		private final JwtAuthenticationConverter jwtAuthenticationConverter;
```

Comment:
Let's leave these changes regarding `JwtAuthenticationConverter` for later.

Reply:
Depending on the choice above this class might change as well.